spec/features/upload_descriptive_metadata_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# This is an integration test (with DSA), because we need to show that the
+# data we submit from here doesn't cause a round trip to MODS error.
```

Comment:
Am I just missing it or is there not a test of what happens when there is a failure?

Reply:
This, being a feature spec, is a full stack spec. They are slow, so we only do them where necessary.  We have more granular specs in `spec/requests` that are not full stack and are faster.